pkg/question/question.go

```diff
@@ -1159,26 +1216,75 @@ func AskInstanceIds(h *ec2helper.EC2Helper, addedInstanceIds []string) (*string,
 	return &answer, err
 }
 
+func AskBootScriptConfirmation(h *ec2helper.EC2Helper, userDefaultBootScript string) string {
+	stringOptions := []string{cli.ResponseYes, cli.ResponseNo}
+	optionsText := yesNoOption + "\n"
+	question := "Confirm to add a filepath to an instance boot script"
```

Comment:
This isn't a question, and the original text was clearer. Could we use that instead? "Add filepath to instance boot script?"

Or, if we need to change it so it doesn't read the same as before, can we use something like "Would you like to add a filepath to the instance boot script?"

```suggestion
	question := "Would you like to add a filepath to the instance boot script?"
```